sql/7_0_3-to-7_0_4_upgrade.sql

```diff
@@ -171,3 +171,34 @@ ALTER TABLE `track_events` DROP INDEX `unique_event_label_url`;
 #IfNotIndex track_events unique_event_label_target
 ALTER TABLE `track_events` ADD UNIQUE `unique_event_label_target` (`event_label`, `event_url`(255), `event_target`(255));
 #EndIf
+
+#IfNotTable care_teams
+CREATE TABLE `care_teams` (
+  `id` int(11) NOT NULL AUTO_INCREMENT,
+  `uuid` binary(16) DEFAULT NULL,
+  `pid` int(11) NOT NULL,
+  `user_id` int(11) NOT NULL,
+  `role` varchar(50) NOT NULL,
+  `facility_id` int(11) NOT NULL,
+  `provider_since` date NOT NULL,
```

Comment:
this date should default to NULL if empty (https://github.com/openemr/openemr/issues/3378#issuecomment-613652036)